src/engine/EventRegistry.ts

```diff
@@ -166,7 +166,9 @@ export class EventRegistry {
         // For each sensor...
         if (sensor.sensorType === "DistanceSensor") {
           const distSensor = <SimDistanceSensor>sensor;
-          const angle = -distSensor.body.getAngle();
+
+          // NOTE!!! We subtract Pi here because we are 180 degrees rotated
+          const angle = -distSensor.body.getAngle() - Math.PI;
```

Comment:
Will this always be the case for every object?

Reply:
It is, since I think the convention we ended up with was X increasing to the right, Z increasing backwards and Y increasing upwards